declarations/Jasper.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Jasper",
+  "documents": {
+    "Privacy Policy": {
+      "fetch": "https://legal.jasper.ai/?_gl=1*gxuhoc*_gcl_au*NDk2Nzk4NzQ1LjE2OTU3NTE4OTI.*_ga*MTU4NTQ1MzEyNy4xNjk1NzUxODk2*_ga_D4P3CS8W5P*MTY5NTc1MTg5Ni4xLjEuMTY5NTc1MTkyNy4yOS4wLjA.#privacy",
+      "select": [
+        "#contract-id-271312 > .contract-header",
+        "#version-646bf007096cf4cb4886fbda-locale-en-us"
+      ],
```

Comment:
```suggestion
      "select": "#contract-body",
```

Reply:
Please note that my suggestion wasn't right 😛 
I used `"select": "#contract-id-271312"` instead.